MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -199,20 +209,31 @@ + (void)destroyPlayer:(NSString *)name {
     [_bindings removeObjectForKey:name];
 }
 
-+ (void)videoChangeForPlayer:(nonnull NSString *)name withVideoData:(nullable MUXSDKCustomerVideoData *)videoData {
++ (void)videoChangeForPlayer:(nonnull NSString *)name withPlayerData:(nullable MUXSDKCustomerPlayerData *)playerData withVideoData:(nullable MUXSDKCustomerVideoData *)videoData {
     if (videoData) {
         MUXSDKPlayerBinding *player = [_viewControllers valueForKey:name];
         if (player) {
             [player dispatchViewEnd];
             [player dispatchViewInit];
             MUXSDKDataEvent *dataEvent = [MUXSDKDataEvent new];
+            [_customerVideoDataStore setVideoData:videoData forPlayerName:name];
             [dataEvent setCustomerVideoData:videoData];
+            if (playerData) {
```

Comment:
Okay so one question here - if you call this with a null playerData, won't this have cleaned itself away and you would end up with a new view that doesn't have an environment key?

Reply:
Or does the store that we have keep that in place on viewInit?

Reply:
The if `if (playerData)`  check should protect us against the `playerData` getting cleared away if someone passes in a `nil` - we only set the player data if it is not `nil`